1.7/administration/high-availability/multi-region.md

```diff
@@ -0,0 +1,38 @@
+---
+post_title: Multiple Regions
+menu_order: 2
+---
+
+DC/OS supports multiple region configurations. This topic describes the setup recommendations and caveats.
+
+**Important:** 
+
+- None of the following multi-AZ setups have been explicitly tested or verified.
```

Comment:
The following multi-region setups are not supported and have not been tested or verified.